src/python/CRABInterface/Regexps.py

```diff
@@ -52,7 +52,7 @@
 RX_JOBID     = re.compile(r"^\d+(-\d+){0,1}$")
 RX_RUNS      = re.compile(r"^\d+$")
 RX_LUMIRANGE = re.compile(r"^\d+,\d+(,\d+,\d+)*$")
-RX_LUMILIST  = re.compile(r"^\d+(,\d+)*$")
+RX_LUMISLIST = re.compile(r"^\d+:?(\d+|None)(,\d+:?(\d+|None))*$")
```

Comment:
Ok yours doesn't really work but how about this:
`^(\d+(,\d+)*)$|^((\d+:(\d+|None))(,\d+:(\d+|None))*)$`